### DISCUSSION TITLE
No extracttoroot.zip?

### DISCUSSION THREAD

**Discussion by solarsX** [QuestCraftPlusPlus/QuestCraft#222] [General]
Its literally not in the downloads so I cant play

**Comment by LameMan0** [comment]
You don't need it

**Reply by mhmatthewhugley** [reply]
They provided at one point a version that was launched and you could just copy all the files and it would work flawlessly. That would be nice to have again.

**Reply by wellsilver** [reply]
> They provided at one point a version that was launched and you could just copy all the files and it would work flawlessly. That would be nice to have again.

I dont know what your referring to but it has never worked flawlessly, just look at the #support channel on discord when it came out

**Comment by wellsilver** [comment]
As of v1.2 the APK installs itself so you need no extracttoroot.zip
